WooCommerce/Classes/POS/Card Present Payments/CardPresentPaymentsModalButtonViewModel.swift

```diff
@@ -0,0 +1,18 @@
+import Foundation
+
+struct CardPresentPaymentsModalButtonViewModel {
+    var title: String
+    var actionHandler: () -> Void
```

Comment:
nit: could they be `let`?

Reply:
Updated in https://github.com/woocommerce/woocommerce-ios/pull/12927/commits/7540fba0e257afb0383b8469eb8ad7d63d366957.